nucypher/characters/control/specifications/fields/datetime.py

```diff
@@ -0,0 +1,12 @@
+from marshmallow import fields
+import maya
+from nucypher.characters.control.specifications.fields.base import BaseField
+
+
+class DateTime(BaseField, fields.Field):
+
+    def _serialize(self, value, attr, obj, **kwargs):
+        return value.iso8601()
+
+    def _deserialize(self, value, attr, data, **kwargs):
+        return maya.MayaDT.from_iso8601(iso8601_string=value)
```

Comment:
I'm unsure, but would a validate function here help? i.e. catch invalid input earlier or is it just redundant since you would use Maya for the validation anyway? What's the code path for `_validate`?

Reply:
Yeah, the _validate runs on the  _output_ Of the _deserialize anyway so it would have failed.  It would be more to check that the date itself met some characteristic.